roles/setup/vars/main.yml

```diff
@@ -5,48 +5,50 @@ general:
 
 # homebrew applications
 applications:
-  - autojump
+#   - autojump
   - git
   - tree
   - httpie
   - curl
   - wget
-  - kubectl
-  - kubernetes-helm
+  - postgresql
+  - rabbitmq
+#   - kubectl
+#   - kubernetes-helm
   - zsh
-  - go
+#   - go
   - vim
-  - hugo
-  - saltstack
-  - neovim
-  - ag 
-  - ctags
+#   - hugo
+#   - saltstack
+#   - neovim
+#   - ag 
+#   - ctags
   - terraform
 
 # homebrew cask applications
 cask_applications:
```

Comment:
This gives me the willies a little. I suppose we should Trust the Pull Request Process on `caskroom`, so I'll allow it for now.